modules/cudf/src/series.ts

```diff
@@ -650,6 +650,25 @@ export class AbstractSeries<T extends DataType = any> {
     return this.__construct(this._col.gather(selection._col));
   }
 
+  /**
+   * Return a copy of this Series.
+   *
+   *
```

Comment:
```suggestion
   *
```

Extremely nit picky, but I think we leave one `*` space inbetween.

Reply:
love a good nit pick, resolved